examples/expsyn/expsyn.py

```diff
@@ -4,14 +4,18 @@
 
 
 import os
+import argparse
 
 import bluepyopt as bpopt
 import bluepyopt.ephys as ephys
 
 
-def main():
+def main(args):
     """Main"""
-    nrn_sim = ephys.simulators.NrnSimulator()
+    if args.sim == 'nrn':
+        sim = ephys.simulators.NrnSimulator()
+    else:
```

Comment:
Not sure why the default here is 'arbor'? In the old code we were just loading the 'nrn' simulator, no?

Reply:
The default for the `--sim` CLI argument is `nrn`,  so by default this script would still use the NrnSimulator.